potential-exploiters/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const createFinding = (deployer: string): Finding => {
+  return Finding.fromObject({
+    name: "Potential Exploiter",
+    description: "A contract was created from a tornado funded account",
+    alertId: "NETHFORTA-28",
+    type: FindingType.Suspicious,
+    severity: FindingSeverity.High,
+    metadata: {
+      deployer: deployer,
+    },
```

Comment:
I would like to see the address of the contract too.

Reply:
Agree 👍